lighthouse-core/scripts/lantern/download-traces.sh

```diff
@@ -1,22 +1,30 @@
 #!/bin/bash
 
+# Downloads the latest golden lantern data from gcloud.
+
 set -e
 
+VERSION="2017-12-06"
+
 DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 LH_ROOT_PATH="$DIRNAME/../../.."
 cd $LH_ROOT_PATH
 
+if [[ -f lantern-data/version ]] && [[ "$VERSION" != "$(cat lantern-data/version)" ]]; then
+  echo "Deleting old lantern data."
```

Comment:
might be nice to give the user a chance to cancel this before deleting, I keep some debug stuff in that folder :D


```suggestion
  echo "Version out of date. About to delete ./lantern-data..."
  echo "Press any key to continue, Ctrl+C to exit"
  read -n 1 -r unused_variable
```

Reply:
This would hang travis ci tho?

Reply:
could throw a `$CI` in there then :)